### PR TITLE
Move kubernetes 1.18 from alpha to stable

### PR DESCRIPTION
**Pull request by mariusv**
Signed-off-by: Marius Voila <myself@mariusv.com>

### CODE REVIEW

**Comment by hakman** [comment]
Thanks for fixing these @mariusv!
/ok-to-test

File: channels/stable [hunk 1]
```diff
@@ -58,7 +58,7 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
-  - range: ">=1.18.0-alpha.1"
+  - range: ">=1.18.0"
```
Comment (hakman) [inline channels/stable:61]: Please do this change in the alpha channel also.

Comment (mariusv) [reply channels/stable:61]: sure thing, in the same PR or should I open a new PR for those?

Comment (hakman) [reply channels/stable:61]: Same PR would be ok.

Comment (mariusv) [reply channels/stable:61]: done

**Pull request merged**